app/views/transactions/_transaction.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="transaction-card">
+  <%= transaction.name %>
+  <p>
+    $ <%= transaction.amount %>
+  </p>
+</div>
```

Comment:
- [ ] In the design it is mentioned to show the transaction creation date. Please add in your UI to follow the design.

### Your design
![Screenshot from 2023-06-03 18-12-21](https://github.com/Talha-Hanif5666/Budget-App/assets/26767383/959bb9ec-f2dc-42d7-ac0d-ab28894b641e)


### Given design
![Screenshot from 2023-06-03 18-13-32](https://github.com/Talha-Hanif5666/Budget-App/assets/26767383/3fcde7cd-d5d1-4574-8bde-5ea2d685932b)

Reply:
Added transaction date✔